weseek/growi
# feat: gw7812 get mentioned users from comment and send notification

https://youtrack.weseek.co.jp/issue/GW-7812
Implement method to get mentioned users from comment and send notification

## Testing Step
- Mention @username in comment editor and submit the comment (Eg: @mudana)
![comment](https://user-images.githubusercontent.com/92426728/170439691-643f3f6f-9887-4aba-bfe5-fff7c3c3ab2d.png)

- Login with mentioned username (@mudana), in-app notification will appear then click it
![profile](https://user-images.githubusercontent.com/92426728/170439981-6a7313ad-6d05-442d-a29a-315c0f65082b.png)

- List of in-app notifications will appear
![notification](https://user-images.githubusercontent.com/92426728/170440153-aae3186b-4248-46f2-91ac-299a4bd75fed.png)

- Test mention in a comment reply. Applied same steps as above
